vcd/resource_vcd_vm_insert_eject_media.go

```diff
@@ -0,0 +1,141 @@
+package vcd
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/vmware/go-vcloud-director/govcd"
+	"log"
+)
+
+func resourceVcdInsertEjectMedia() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceVcdMediaInsert,
+		Delete: resourceVcdMediaEject,
+		Read:   resourceVcdVmInsertedMediaRead,
+
+		Schema: map[string]*schema.Schema{
+			"vdc": {
+				Type:     schema.TypeString,
+				Required: false,
+				Optional: true,
+				ForceNew: true,
+			},
+			"org": {
+				Type:     schema.TypeString,
+				Required: false,
+				Optional: true,
+				ForceNew: true,
+			},
+			"catalog": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "catalog name where to find media file",
+			},
+			"name": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "media name to use",
+			},
+			"vapp_name": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "vapp to use",
```

Comment:
Please change "vapp" to "vApp" and "vm" to "VM" when using in comments.

Reply:
fixed